lib/webinspector/index.js

```diff
@@ -7,20 +7,31 @@ import LengthBasedSplitter from '../util/transformer/length-based-splitter';
 
 const WEB_INSPECTOR_SERVICE_NAME = 'com.apple.webinspector';
 
+const PARTIAL_MESSAGE_SUPPORT_DEPRECATION_VERSION = 11;
+
 class WebInspectorService {
 
-  constructor (socketClient) {
+  constructor (version, socketClient) {
```

Comment:
it would be really handy to have some documentation here, because it's completely unclear what the expected types of the arguments are

Reply:
I get that. Will explain it